injector/cpu_pressure_test.go

```diff
@@ -80,18 +90,26 @@ var _ = Describe("Failure", func() {
 	})
 	Describe("injection", func() {
```

Comment:
It seems we were not asserting our mock `On/Return` setup where effectively called 😬 
That is usually not a good thing because you expect such to be validated.
I discovered this because:
```go
stresserManager.On("CoresToBeStressed").Return(cpuset.NewCPUSet(0, 1))
```

Is part of our setup but never called 😬 Hence I would have expected a test failure that wasn't there.

Here is a proposal to add all of our mock assertions for all tests (tested locally):
```suggestion

	AfterEach(func() {
		manager.AssertExpectations(GinkgoT())
		stresser.AssertExpectations(GinkgoT())
		stresserManager.AssertExpectations(GinkgoT())
		cgroupManager.AssertExpectations(GinkgoT())
		ctn.AssertExpectations(GinkgoT())
	})

	Describe("injection", func() {
```

You will also need to remove the following line from the setup:
```go
cgroupManager.On("Read", "cpuset", "cpuset.cpus").Return("0-1", nil)
```

This is no longer expected as we are now mocking this